i18n/locales/source/dashboard/block_categories.yml

```diff
@@ -0,0 +1,32 @@
+---
+en:
+  data:
+    block_categories:
+      Actions: Actions
+      Advanced: Advanced
+      Behaviors: Behaviors
+      Brushes: Brushes
+      Color: Color
+      Comment: Comment
+      Comments: Comments
+      Conditionals: Conditionals
+      Dancers: Dancers
+      Default: Default
+      Events: Events
+      Functions: Functions
+      'Functions ': 'Functions '
+      Game/Score: Game/Score
+      Games/Score: Games/Score
+      Groups: Groups
+      Input: Input
+      Locations: Locations
+      Logic: Logic
+      Loops: Loops
+      Math: Math
+      Numbers: Numbers
+      Properties: Properties
+      Sprites: Sprites
+      Text: Text
+      Variables: Variables
+      'Variables ': 'Variables '
+      World: World
```

Comment:
this "new" file is a result of https://github.com/code-dot-org/code-dot-org/pull/32095